sidebars/general.js

```diff
@@ -31,8 +31,19 @@ const sidebars = {
                     label: 'Process',
                     type: 'category',
                     items: [
-                        'development/process/peer-review',
-                        'development/process/integration-review',
+                            'development/process/peer-review',
+                            'development/process/integration-review',
+                            'development/process/testing',
+                            {
+                                type: 'doc',
+                                id: 'development/process/testing-integrated-issues',
+                                className: 'docusaurus-hidden',
```

Comment:
I'm really not sure that I like the hidden documents. I had to grep for how to find these and I think that, at the moment, there's nothing stopping us from making them nest underneath the "Testing" component.

Reply:
I've created the Testing category and added the "Integrated issues" and "QA" there. 

![testingsidebar](https://user-images.githubusercontent.com/900389/163143563-64d63282-31a0-4e1d-8ffe-79a4f4b76cde.png)

I think we should re-think the navbar because I can't see the benefit of having "Process & policies" under the same tab (it seems better to have two separate items for them). Thoughts?  I think it should be treated as a separate issue, but raising it here...